conjur/errors.py

```diff
@@ -168,3 +168,24 @@ class KeyringWrapperSetError(KeyringWrapperGeneralError):
     def __init__(self, message: str = ""):
         self.message = message
         super().__init__(self.message)
+
+class HttpError(Exception):
+    """ Base exception for general HTTP failures """
+    def __init__(self, message: str = "HTTP request failed", response: str = ""):
+        self.message = message
+        self.response = response
```

Comment:
I think you can omit the variables assignments since it happens in the base class
```suggestion
```

Reply:
done